src/commands/synthetics/errors.ts

```diff
@@ -21,3 +24,33 @@ export class CiError extends Error {
 }
 
 export class CriticalError extends CiError {}
+
+export const handleCiError = (error: CiError, reporter: MainReporter) => {
+  switch (error.code) {
+    case 'NO_TESTS_TO_RUN':
+      reporter.log('No test to run.\n')
+      break
+    case 'MISSING_APP_KEY':
+      reporter.error(`Missing ${chalk.red.bold('DATADOG_APP_KEY')} in your environment.\n`)
+      break
+    case 'MISSING_API_KEY':
+      reporter.error(`Missing ${chalk.red.bold('DATADOG_API_KEY')} in your environment.\n`)
+      break
+    case 'POLL_RESULTS_FAILED':
+      reporter.error(`\n${chalk.bgRed.bold(' ERROR: unable to poll test results ')}\n${error.message}\n\n`)
+      break
+    case 'TUNNEL_START_FAILED':
+      reporter.error(`\n${chalk.bgRed.bold(' ERROR: unable to start tunnel')}\n${error.message}\n\n`)
+      break
+    case 'TRIGGER_TESTS_FAILED':
+      reporter.error(`\n${chalk.bgRed.bold(' ERROR: unable to trigger tests')}\n${error.message}\n\n`)
+      break
+    case 'UNAVAILABLE_TEST_CONF':
+      reporter.error(
+        `\n${chalk.bgRed.bold(' ERROR: unable to obtain test configurations with search query ')}\n${error.message}\n\n`
+      )
+      break
+    case 'UNAVAILABLE_TUNNEL_CONF':
+      reporter.error(`\n${chalk.bgRed.bold(' ERROR: unable to get tunnel configuration')}\n${error.message}\n\n`)
+  }
+}
```

Comment:
This might be counter-intuitive, but given that this function uses the reporter and `chalk`, I think like we want it in, or close to, the cli part, not in the `errors` file which is the responsibility of `run-test` which should be CLI agnostic.

Reply:
I've moved the reportCiError function back to the cli section